cloud-config/google-cloud/startup-script.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+if [ -f /root/aggregate.version ]; then
+  echo "System already configured. Skipping script"
+  exit 0
+fi
+
+cat << EOF > /root/aggregate-config.json
+{
+  "home": "/root",
+  "jdbc": {
+    "host": "127.0.0.1",
+    "port": 5432,
+    "db": "aggregate",
+    "schema": "aggregate",
+    "user": "aggregate",
+    "password": "aggregate"
+  },
+  "security": {
+    "hostname": "foo.bar",
+    "forceHttpsLinks": true,
+    "port": 80,
+    "securePort": 443,
+    "checkHostnames": false
+  },
+  "tomcat": {
+    "uid": "tomcat8",
+    "gid": "tomcat8",
+    "webappsPath": "/var/lib/tomcat8/webapps"
+  }
+}
+EOF
+
+add-apt-repository -y universe
+add-apt-repository -y ppa:certbot/certbot
+apt-get -y update
+apt-get -y install \
+ zip \
+ unzip \
+ wget \
+ curl \
+ tomcat8 \
+ postgresql-10 \
+ openjdk-8-jdk-headless \
+ nginx \
+ software-properties-common \
+ python-certbot-nginx
```

Comment:
This step failed for me trying to configure tzdata, which I think comes with certbot. Log at http://dpaste.com/1CCQFWQ.

According to https://askubuntu.com/questions/909277, it might work if you sent the `DEBIAN_FRONTEND` variable to `noninteractive`. https://stackoverflow.com/questions/8671308/non-interactive-method-for-dpkg-reconfigure-tzdata might also help.

Reply:
How did you get the logs of the startup script?

Reply:
syslog! nevermind :)